src/gpodder/minidb.py

```diff
@@ -193,9 +193,9 @@ def get(self, class_, **kwargs):
     class Person(object):
         __slots__ = {'username': str, 'id': int}
 
-        def __init__(self, username, id):
+        def __init__(self, username, userid):
```

Comment:
It would be better if 'userid' was 'user_id' or 'uid'. There are several instances in this file.

Reply:
I changed it to 'uid'.